Fill.py

```diff
@@ -228,7 +228,7 @@ def distribute_items_restrictive(world: MultiWorld) -> None:
         items_counter = Counter([location.item.player for location in world.get_locations() if location.item])
         locations_counter = Counter([location.player for location in world.get_locations()])
         items_counter.update([item.player for item in unplaced])
-        locations_counter.update([location.player for location in unfilled])
+        locations_counter.update([location.player for location in defaultlocations])
```

Comment:
Because its causing an exception as seen here https://discord.com/channels/731205301247803413/763938462092886067/975457113725866064
`unfilled` is a list of string which wont have a `player` attribute

Reply:
Ah, that can use some updating in general, doing that here: https://github.com/ArchipelagoMW/Archipelago/pull/526